src/app/src/components/TitledDrawer.jsx

```diff
@@ -50,7 +50,11 @@ const TitledDrawer = ({
             </div>
             <Typography className={classes.drawerTitle}>{title}</Typography>
             <Typography className={classes.drawerSubtitle}>
-                {subtitle}
+                <>
+                    {subtitle}
+                    <br />
+                    Latitude Longitude
```

Comment:
1. I think this text should be passed along with the `subtitle` prop. According to the UX mockups, it seems that the comma is missing between the words. -> ` Latitude, Longitude`.
2. Please don't forget to verify other places where the TitledDrawer component is used.
**Issue:**
<img width="1563" alt="Screenshot 2024-02-16 at 09 54 09" src="https://github.com/opensupplyhub/open-supply-hub/assets/58369841/1016f2e5-a44f-4f2f-a24a-746196816c7a">

Reply:
Fixed